docs/source/index.rst

```diff
@@ -3,10 +3,16 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-PyTorch Lightning Documentation
-===============================
+PyTorch Lightning
+=================
```

Comment:
Let's also add a proper introduction to the front page in a follow up? I can help :) 

Right now it starts with tutorials and the first thing that shows up is an introduction to PyTorch which is not right.

Reply:
@tchaton  :)

Reply:
Yeah, would be good to do this in a follow-up :smiley: